src/analyzer/plugins/analyzerkeyfinder.cpp

```diff
@@ -0,0 +1,123 @@
+#include "analyzer/plugins/analyzerkeyfinder.h"
+
+#include "analyzer/constants.h"
+#include "util/assert.h"
+#include "util/math.h"
+
+using mixxx::track::io::key::ChromaticKey;
+using mixxx::track::io::key::ChromaticKey_IsValid;
+
+namespace mixxx {
+
+AnalyzerKeyFinder::AnalyzerKeyFinder()
+        : m_pKeyFinder(nullptr),
```

Comment:
The default constructor works fine, no initialization required

Reply:
Does a new analyzer get instantiated for every track, or is a single analyzer instance reused by calling `initialize`?

Reply:
Okay, it looks like every Track gets its own set of analyzer instances, so using normal class member objects and initializing them in the initializer list is fine.